packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/DdSessionReplayImplementation.kt

```diff
@@ -27,9 +29,11 @@ class DdSessionReplayImplementation(
      * @param defaultPrivacyLevel The privacy level used for replay.
      */
     fun enable(replaySampleRate: Double, defaultPrivacyLevel: String, promise: Promise) {
+        val sdkCore = Datadog.getInstance() as FeatureSdkCore
+        val logger = sdkCore.internalLogger
```

Comment:
I'd like to avoid passing the `logger` down every class until we really need it.

What do you think about putting this in a DatadogSDKWrapper? (Maybe we can merge the one Session Replay SDK Wrapper with it)
Then using Dependency Injection we can still have it as an constructor argument for classes that need them?

Reply:
I don't know if this is still needed, but let's discuss

Reply:
Indeed it looks like it's no longer needed 👍